src/stream.js

```diff
@@ -21,8 +26,63 @@ module.exports.handler = async (event) => {
 	delete(groupedRecords['deleteMe']); // delete bad events
 
 	for (const records of Object.entries(groupedRecords)) {
-		const latestRecord = records.pop();
+		logger.debug({records}, 'Records');
+		const latestRecord = records.pop().pop();
```

Comment:
Why `pop().pop()`?

Reply:
I should probably use lodash `_.last` Basically, just the way lodash setup the groupby array.

Reply:
_.last and pop have the same outcome. I'll need to do two lasts for it to work too.

```
"latestRecord": [
     "date-channel-Index",
     [
        {
            "date": "2019-11-25",
            "channel": "C98087QMN",
            "questionText": "New Question",
            "users": {}
        }
    ]
]
```
it is decoding this.